SpringProject/src/main/java/com/gradproject2019/conferences/data/ConferenceRequestDto.java

```diff
@@ -0,0 +1,123 @@
+package com.gradproject2019.conferences.data;
+
+import com.gradproject2019.conferences.persistance.Conference;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotNull;
+import java.time.Instant;
+
+@Validated
+public class ConferenceRequestDto {
+    private Long id; // FE should be giving BE id = null
```

Comment:
This comment is only relevant for saving a conference, probably shouldn't be here